code/game/objects/items/plushes.dm

```diff
@@ -9,11 +9,52 @@
 	var/list/squeak_override //Weighted list; If you want your plush to have different squeak sounds use this
 	var/stuffed = TRUE //If the plushie has stuffing in it
 	var/obj/item/grenade/grenade //You can remove the stuffing from a plushie and add a grenade to it for *nefarious uses*
+	//--love ~<3--
+	var/male_chance = 1
+	var/female_chance = 1
+	var/obj/item/toy/plush/lover
+	var/obj/item/toy/plush/partner
+	var/obj/item/toy/plush/plush_child
+	var/obj/item/toy/plush/paternal_parent	//who initiated creation
+	var/obj/item/toy/plush/maternal_parent	//who owns, see love()
+	var/list/scorned	//who the plush hates
+	var/heartbroken = FALSE
+	var/vowbroken = FALSE
+	var/young = FALSE
+	var/mood_message = null
+	var/list/love_message
+	var/list/heartbroken_message
+	var/list/vowbroken_message
+	var/list/child_message
+	var/normal_desc
+	//--end of love :'(--
 
 /obj/item/toy/plush/Initialize()
 	. = ..()
 	AddComponent(/datum/component/squeak, squeak_override)
 
+/obj/item/toy/plush/New()
+	//time to assume Pinocchio's gender
+	if(male_chance + female_chance <= 0)	//if neither genders are present
+		gender = NEUTER
+	else if(male_chance <=0)	//if cannot be male
+		gender = FEMALE
+	else if(female_chance <= 0)	//if cannot be female
+		gender = MALE
+	else if(rand(0, (male_chance+female_chance)) > male_chance)	//if a random number from 0 to total of both chances > one chance (here it's male)...
+		gender = FEMALE	//... gender will be female (since the check is "not male")
+	else	//... else the number landed on the other chance
+		gender = MALE
+
+	normal_desc = desc
+
+	love_message += ("\n\The [src] is so happy, \he could rip a seam!")
+	heartbroken_message += ("\n\The [src] looks so sad.")
+	vowbroken_message += ("\n\The [src] lost \his ring...")
+	child_message += ("\n\The [src] can't remember what sleep is.")
+
+	return ..()
+
 /obj/item/toy/plush/Destroy()
```

Comment:
null all the vars you added at this stage, and if they have a lover, null it's lover var pointing back to this object best if you have an unset_lover proc, so you can just lover.unset_lover(src)

Reply:
also use the QDEL_LIST operator to clear the scorned list

Reply:
Good point. Will do!